### DISCUSSION TITLE
Need help in selecting mutiple things from a dropdown in my nuxt 3 project

### DISCUSSION THREAD

**Discussion by Calvinasty** [nuxt/nuxt#22586] [Questions]
Need help implementing the mutliselect or vue select in my nuxt 3 project. I keep getting Cannot read props of undefined (reading 'options') after in install in the code

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction) or what you tried so far? 🙏
